apps/ledger-live-desktop/src/renderer/components/WebPTXPlayer/CustomHandlers.ts

```diff
@@ -1,21 +1,27 @@
-import React, { useMemo } from "react";
+import React, { useCallback, useMemo } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { WalletAPICustomHandlers } from "@ledgerhq/live-common/wallet-api/types";
 import {
   handlers as exchangeHandlers,
   ExchangeType,
 } from "@ledgerhq/live-common/wallet-api/Exchange/server";
 import trackingWrapper from "@ledgerhq/live-common/wallet-api/Exchange/tracking";
-import { AccountLike, Operation } from "@ledgerhq/types-live";
+import { Operation } from "@ledgerhq/types-live";
 import { track } from "~/renderer/analytics/segment";
 import { currentRouteNameRef } from "~/renderer/analytics/screenRefs";
 import { closePlatformAppDrawer, openExchangeDrawer } from "~/renderer/actions/UI";
 import { WebviewProps } from "../Web3AppWebview/types";
 import { context } from "~/renderer/drawers/Provider";
 import WebviewErrorDrawer from "~/renderer/screens/exchange/Swap2/Form/WebviewErrorDrawer";
 import { platformAppDrawerStateSelector } from "~/renderer/reducers/UI";
+import { flattenAccountsSelector } from "~/renderer/reducers/accounts";
+import { store } from "~/renderer/init";
 
-export function usePTXCustomHandlers(manifest: WebviewProps["manifest"], accounts: AccountLike[]) {
+function getAccounts() {
+  return flattenAccountsSelector(store.getState());
+}
```

Comment:
It's quite strange to do that directly instead of using a the selector with `useSelector`
It's seems we're not addressing the root cause of the issue, a change to the accounts should be reflected to the handlers